src/libs/styled/useColorScheme.tsx

```diff
@@ -1,38 +1,55 @@
-import { ColorSchemeName, Appearance } from 'react-native'
+import { useColorScheme as useSystemColorScheme } from 'react-native'
 
+import { useFeatureFlag } from 'libs/firebase/firestore/featureFlags/useFeatureFlag'
+import { RemoteStoreFeatureFlags } from 'libs/firebase/firestore/types'
 import { createStore } from 'libs/store/createStore'
 
-export type ColorScheme = 'dark' | 'light'
+export enum ColorSchemeEnum {
+  LIGHT = 'light',
+  DARK = 'dark',
+  SYSTEM = 'system',
+}
 
-type State = { colorScheme: ColorScheme }
+export type ColorScheme = ColorSchemeEnum.LIGHT | ColorSchemeEnum.DARK
+export type ColorSchemeFull = ColorSchemeEnum
 
-const DEFAULT_COLOR_SCHEME = 'light'
+type State = { colorScheme: ColorSchemeFull }
 
-const defaultState: State = {
-  colorScheme: DEFAULT_COLOR_SCHEME,
-}
+const DEFAULT_COLOR_SCHEME: ColorSchemeFull = ColorSchemeEnum.LIGHT
+const defaultState: State = { colorScheme: DEFAULT_COLOR_SCHEME }
 
 const colorSchemeStore = createStore({
   name: 'colorScheme',
   defaultState,
   actions: (set) => ({
-    init: () => set({ colorScheme: Appearance.getColorScheme() || DEFAULT_COLOR_SCHEME }),
-    setColorScheme: ({ colorScheme }: { colorScheme: ColorSchemeName }) => {
-      set({ colorScheme: colorScheme || DEFAULT_COLOR_SCHEME })
+    setColorScheme: ({ colorScheme }: { colorScheme: ColorSchemeFull }) => {
+      set({ colorScheme })
```

Comment:
IMHO, on pourrait stocker la valeur `light` ou `dark` et pas `SYSTEM`

comme ça, on pourrait utiliser ce store dans devoir passer par le hook

```suggestion
      set({ colorScheme })
```

Reply:
Ça ne fonctionne pas, en tout cas j'ai pas réussi.. :(